fitz/fitz.i

```diff
@@ -14412,6 +14419,63 @@ struct Tools
         }
 
 
+        FITZEXCEPTION(set_word_delimiters, !result)
+        %pythonprepend set_word_delimiters %{
+        """Set characters to be word delimiters."""
+        if delims == None:
+            delims = []
+        if not hasattr(delims, "__getitem__") or len(delims) > 64:
+            raise ValueError("bad delimiter value(s)")
+
+        try:
+            delims = set([ord(c) for c in delims if ord(c) > 32])
+        except:
```

Comment:
Minor thing, but doing `except Exception:` ignores Ctrl-C etc, allowing user to interrupt.